.travis.yml

```diff
@@ -1,14 +1,8 @@
 sudo: false
 language: go
 go:
-  - 1.4
-  - 1.5
-  - 1.6
-  - 1.7
-  - tip
-matrix:
-  allow_failures:
-    - go: tip
+  - 1.9
```

Comment:
Does Travis support 1.9 yet? The last time I checked, they still weren't supporting 1.9.